hack/.ci/run-e2e-gke.sh

```diff
@@ -53,109 +62,153 @@ function kubectl_create {
-          - /usr/bin/bash
-          - -euExo
-          - pipefail
-          - -O
-          - inherit_errexit
-          - -c
-          args:
-          - |
-            sysctl fs.aio-max-nr=0xffffffff
-
-            sleep infinity &
-            wait
-        nodeSelector:
-          scylla.scylladb.com/node-type: scylla
+        selector:
+          matchLabels:
+            app.kubernetes.io/name: sysctl
+        template:
+          metadata:
+            labels:
+              app.kubernetes.io/name: sysctl
+          spec:
+            containers:
+            - name: sysctl
+              securityContext:
+                privileged: true
+              image: "${SO_IMAGE}"
+              imagePullPolicy: IfNotPresent
+              command:
+              - /usr/bin/bash
+              - -euExo
+              - pipefail
+              - -O
+              - inherit_errexit
+              - -c
+              args:
+              - |
+                sysctl fs.aio-max-nr=0xffffffff
+
+                sleep infinity &
+                wait
+            nodeSelector:
+              scylla.scylladb.com/node-type: scylla
 EOF
-kubectl -n=default rollout status daemonset/sysctl
+    kubectl -n=default rollout status daemonset/sysctl
 
-kubectl apply --server-side -f=./pkg/api/scylla/v1alpha1/scylla.scylladb.com_nodeconfigs.yaml
-kubectl wait --for=condition=established crd/nodeconfigs.scylla.scylladb.com
+    kubectl apply --server-side -f=./pkg/api/scylla/v1alpha1/scylla.scylladb.com_nodeconfigs.yaml
+    kubectl wait --for=condition=established crd/nodeconfigs.scylla.scylladb.com
 
-if [[ "${SO_DISABLE_NODECONFIG}" == "true"  ]] || [[ -z "${SO_NODECONFIG_PATH}" ]]; then
-  echo "Skipping NodeConfig creation"
-else
-  kubectl_create -f="${SO_NODECONFIG_PATH}"
-  kubectl_create -n=local-csi-driver -f=./hack/.ci/manifests/namespaces/local-csi-driver/
+    if [[ "${SO_DISABLE_NODECONFIG}" == "true"  ]] || [[ -z "${SO_NODECONFIG_PATH}" ]]; then
+      echo "Skipping NodeConfig creation"
+    else
+      kubectl_create -f="${SO_NODECONFIG_PATH}"
+      kubectl_create -n=local-csi-driver -f=./hack/.ci/manifests/namespaces/local-csi-driver/
+    fi
+
+    kubectl create namespace e2e --dry-run=client -o=yaml | kubectl_create -f=-
+    kubectl create clusterrolebinding e2e --clusterrole=cluster-admin --serviceaccount=e2e:default --dry-run=client -o=yaml | kubectl_create -f=-
+  )
+}
+
+for i in "${!kubeconfigs[@]}"; do
+  setup "${kubeconfigs[$i]}" &
+  setup_bg_pids["${i}"]=$!
+done
+
+setup_failed=0
+for pid in "${setup_bg_pids[*]}"; do
+  wait "${pid}" || setup_failed=1
+done
+
+if [ ${setup_failed} -eq 1 ]; then
+  echo "Error setting up clusters" > /dev/stderr
+  exit 2
 fi
 
-kubectl create namespace e2e --dry-run=client -o=yaml | kubectl_create -f=-
-kubectl create clusterrolebinding e2e --clusterrole=cluster-admin --serviceaccount=e2e:default --dry-run=client -o=yaml | kubectl_create -f=-
+KUBECONFIG="${kubeconfigs[0]}"
+export KUBECONFIG
 
```

Comment:
why isn't the operator installed in the other DC as well?

Reply:
looks like the next couple of lines should be a part of `setup` func as well, thanks